### PR TITLE
kubetail: install completions

### PR DESCRIPTION
**Pull request by yurrriq**
###### Motivation for this change

Install completions for kubetail. N.B. I tested fish completions. YMMV with bash and zsh.. I've seen some confusion re: {ba,z}sh on Nix lately..

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by c0bw3b** [comment]
@kalbasit you okay with this as maintainer?
Seems perfectly sound to me but just want to be sure.

**Comment by kalbasit** [comment]
@c0bw3b it LGTM. Let me test out bash and zsh. I'll merge it if it works well.

**Comment by yurrriq** [comment]
Thanks, y'all!

File: pkgs/applications/networking/cluster/kubetail/default.nix
Comment (kalbasit) [inline pkgs/applications/networking/cluster/kubetail/default.nix]: @yurrriq the install commands are not consistent and make things confusing, can you swap the input/output in the second command? Otherwise LGTM.

**Review by kalbasit — CHANGES_REQUESTED** [review]

**Review by kalbasit — APPROVED** [review]
Looks great! Thank you!

**Comment by yurrriq** [comment]
:tada: Thanks for the quick response!

**Pull request merged**
